File: conditionals/oldEnough.py
age = int(input("How old are you?\n"))
voter = "yes"
if age == 999:
    print("How are you still alive?")
elif age >= 18 and voter == "yes":
    print("You are able to vote!\n")
elif age >= 16:
    print("You are old enough to drive\n")
elif age >= 15:
    print("You are old enough to get a learners permit!\n")
elif age >= 5:
    print("You are old enough to go to school!\n")
else:
    print("You are to young to go to school :(\n")

File: conditionals/oldEnough.c
#include <stdio.h>
#include <stdbool.h>
int age = 4;
bool vote = false;
int main(void){
    if (age >= 18 && vote == true){
        printf("You can vote!\n");
    }else if (age >= 16){
        printf("You are old enough to drive!\n");
    }else if (age == 15){
        printf("You are old enough to get a learners permit!\n");
    }else if (age >= 5){
        printf("You are old enough to go to school!\n");
    }else{
        printf("Sorry, you are not old enough to go to school :(\n");
    }
    return 0;
}